backend/ianalyzer/common_settings.py

```diff
@@ -4,10 +4,19 @@
 '''
 
 import os
+import sentry_sdk
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+if SENTRY_DSN := os.getenv('SENTRY_DSN'):
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        enable_tracing=True,
+        sample_rate=1.0,
+        traces_sample_rate=1.0
+    )
+
```

Comment:
Sorry for yet another change request, but will this mean we cannot override the values that are set here? Perhaps the best place for this would be in `backend.manage.main()`, right after the `os.environ.setdefault...`. Then the `common_settings` would have:
```
SENTRY_DSN := os.getenv('SENTRY_DSN')
SENTRY_ENABLE_TRACING=True
SENTRY_SAMPLE_RATE = 1.0
SENTRY_TRACES_SAMPLE_RATE = 1.0
```

And `manage.py`:
```
if SENTRY_DSN:
   sentry_sdk.init(
      dsn=SENTRY_DSN,
      enable_tracing=SENTRY_ENABLE_TRACING,
      sample_rate=SENTRY_SAMPLE_RATE,
      traces_sample_rate=SENTRY_TRACES_SAMPLE_RATE
)
```
That way, we would be able to set different values for `enable_tracing` etc. per instance, but keep a Sentry init for all instances.